app/helpers/template_helpers.py

```diff
@@ -42,6 +42,90 @@ def get_page_header_context(language, theme):
     return context.get(theme)
 
 
+def get_footer_context(
+    language_code, static_content_urls, sign_out_url, theme="census"
```

Comment:
We are defaulting theme to census in more than one place. Looking at the code, I think we could change line 142 to:
```python
theme = cookie_session.get("theme", DEFAULT_THEME)
```

And define `DEFAULT_THEME` at the top of the file.

Then we can remove the defaulting from the page header and footer context calls.